docs/tempo/website/configuration/manifest.md

```diff
@@ -18,7 +18,6 @@ go run ./cmd/tempo --storage.trace.backend=local --storage.trace.local.path=/tmp
 
 ```yaml
 target: all
-multitenancy_enabled: true
```

Comment:
This is the result of running the listed command again. Should updating this segment be part of our release procedure?

Reply:
I like that idea! 

If we don't add something like that... most people (at least me:)) will forget to update that section on PRs.

Reply:
Done :+1: